RepoCleanup/Functions/SetupNewServiceOwnerFunction.cs

```diff
@@ -42,21 +41,31 @@ public static async Task Run()
             {
                 Console.WriteLine($"Created all default teams for {org.Fullname}");
             }
-            
-            // Create default datamodels repository
+
+            // Create default repositories
+            var isDatamodelRepoCreated = await CreateRepo(giteaService, org, "datamodels", true);
+            var isContentRepoCreated = await CreateRepo(giteaService, org, "content", false);
+
+            if (isDatamodelRepoCreated && isContentRepoCreated)
+            {
+                Console.WriteLine("Done setting up new service owner in Gitea!");
+            }
+        }
+
+        private static async Task<bool> CreateRepo(GiteaService giteaService, Organisation org, string repoName, bool preFixRepoNameWithOrg)
+        {
             var createRepoForOrgsCommandHandler = new CreateRepoForOrgsCommandHandler(giteaService);
-            var numberOfReposCreated = await createRepoForOrgsCommandHandler.Handle(new CreateRepoForOrgsCommand(new List<string>() { org.Username }, "datamodels", true));
+            var numberOfReposCreated = await createRepoForOrgsCommandHandler.Handle(new CreateRepoForOrgsCommand([org.Username],  repoName, preFixRepoNameWithOrg));
```

Comment:
Should not this parameter always be true for repos `org-datamodels` and `org-content`?
Then we could remove the `preFixRepoNameWithOrg` parameter from `CreateRepo`.

```suggestion
            var numberOfReposCreated = await createRepoForOrgsCommandHandler.Handle(new CreateRepoForOrgsCommand([org.Username],  repoName, true));
```